examples/Bickley_jet_shallow_water.jl

```diff
@@ -76,43 +69,51 @@ model = ShallowWaterModel(
 
 # ## Background state and perturbation
 #
-# The background velocity ``\overline{u}`` and free-surface ``\overline{\eta}`` are chosen to 
-# represent a geostrophically balanced Bickely jet with maximum speed of ``U`` and 
-# maximum free-surface deformation of ``Δη``, i.e.,
+# The background velocity ``ū`` and free-surface ``η̄`` are chosen to represent a 
+# geostrophically balanced Bickely jet with maximum speed of ``U`` and maximum 
+# free-surface deformation of ``Δη``, i.e.,
 #
 # ```math
-# \overline{\eta}(y) = - Δη \tanh(y) , \\
-# \overline{u}(y) = U \mathrm{sech}^2(y) .
+# \begin{aligned}
+# η̄(y) & = - Δη \tanh(y) , \\
+# ū(y) & = U \mathrm{sech}^2(y) .
+# \end{aligned}
 # ```
-# 
-# The vorticity of the background state is ``Ω = - \partial_y \overline{u} = 2 U \mathrm{sech}^2(y) \tanh(y)``.
 #
-# Linear stability theory predicts that for the particular parameters that we consider here,
-# the growth rate for the maximum growth rate should be ``0.14``.
+# The total height of the fluid is ``h = L_z + \eta``. Linear stability theory predicts that 
+# for the parameters we consider here, the growth rate for the most unstable mode that fits 
+# our domain is approximately ``0.14``.
```

Comment:
@francispoulin is 0.14 the growth rate for  `k = 1`?

Reply:
Yes, to two significant digits, it is `0.14`.

Reply:
is it 0.138 if we keep 3 significant? because that's really very close to the computed one

Reply:
More like `0.139`